### ISSUE TITLE
Is this plugin abbandoned?

### ISSUE THREAD

**Issue by Naqsaq** [lprhodes/homebridge-broadlink-rm#626]
Many seem to be worried that this plugin is abbandoned while issues are piling up and the never devices aren't getting support (although there few workarounds for them).

Is this really the case? Would be quite useful info for new users committing to the Broadlink HW.

**Comment by l3nticular** [comment]
Check out the forks; I think there's a lot of active work going on, but it's not getting merged back here.  I don't know if the PRs just aren't getting submitted or if perhaps one of the forks should be the "default".

**Comment by Naqsaq** [comment]
Yeah, there are forks to get most of the things working. But then again, it's not a premanent solution. E.g. as the fan accessory is broken in the latest release, we're forced to use 3.6.18 to begin with...

**Comment by SmolSoftBoi** [comment]
The forks also aren't bringing all the issues being solved together either.
